provider/kannel/kannel.go

```diff
@@ -0,0 +1,65 @@
+package kannel
+
+import (
+	"fmt"
+	"net/url"
+	"net/http"
+	"time"
+
+	"github.com/messagebird/sachet"
+)
+
+//KannelConfig configuration struct for Kannel Client
+type KannelConfig struct {
+	URL 	string `yaml:"url"`
+	User  string `yaml:"username"`
```

Comment:
I think you are missing some `gofmt` in there. Please be sure you run it on all files 🙏